### ISSUE TITLE
[Idea] Path directives highlight

### ISSUE THREAD

**Issue by myfonj** [lishu/vscode-svg#9]
It would be neat to have `path`/`glyph` `d` attribute value highlighted so that commands, coordinates, lengths, angles and flags would be more apparent.

Mock (pardon colours):
![image](https://user-images.githubusercontent.com/1761395/37116584-013f3534-224f-11e8-9a2b-4070ccf7e632.png)

I assume it should be possible; the only danger I see are omittable consecutive commands.

[mock.html.txt](https://github.com/lishu/vscode-svg/files/1790787/mock.html.txt)

**Comment by lishu** [comment]
Yes, this is a good idea, but it may be an inefficient process. I will include it in future plans.

是的，这是个好主意，但这可能是个不高效的程序，我会将它列入将来的计划中。
